Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -687,7 +596,7 @@ def __hash__(self):
     def __eq__(self, other):
         # XXX: comparing NaN with different payload
         return (isinstance(other, ComparableInstance)
-                and domain_eq(self.domain, other.domain)
+                and (self.domain == other.domain)
```

Comment:
This now compares metas as well (before we didn't). Intentional?

Reply:
It was requested in the original issue, that domains match completely, when doing comparisons over rows. This is a remnant of that, but as we later decided against the idea, i should have  reverted it. Good catch.